scripts/tests/upgrade-test.sh

```diff
@@ -26,11 +26,6 @@ for version in "${versions[@]}"; do
   gatewayd_paths+=("$(nix_build_binary_for_version 'gatewayd' "$version")")
 done
 
-# Add current binaries from PATH
-fedimintd_paths+=("fedimintd")
-fedimint_cli_paths+=("fedimint-cli")
-gatewayd_paths+=("gatewayd")
```

Comment:
I think we want the ability to test upgrades to current master (which I assume these lines are about). Maybe there could be a special-cased version `current` that turns these on/off as the last step?

Reply:
Good call 👍 

I've created an issue for fixing the error with discovering common core api versions when upgrading to `master`.

https://github.com/fedimint/fedimint/issues/4960